src/store/ViewStore.js

```diff
@@ -92,6 +92,7 @@ export default class ViewStore {
                 this.mainStore.studies.updateActiveStudies();
                 if (this.loader) { this.loader.hide(); }
                 this.mainStore.state.setChartIsReady(true);
+                this.mainStore.state.setChartGranularity(getIntervalInSeconds(ViewStore.views[idx].layout));
```

Comment:
@shayan-binary it'd be to break the line or assign the granularity in seconds to a const variable and pass it to `setChartGranularity`